addOns/websocket/src/main/java/org/zaproxy/zap/extension/websocket/ui/PopupIncludeWebSocketContextMenu.java

```diff
@@ -58,6 +58,13 @@ protected ExtensionPopupMenuItem createPopupIncludeInContextMenu() {
         return new PopupIncludeWebSocketInContextMenu();
     }
```

Comment:
Do we need the older override now? I couldn't find usages of this method in core anymore.

Reply:
In theory we shouldn't need but… yes, better deprecate that method now.

Reply:
Old method removed.